site/docs/webrtc/voice-iw.md

```diff
@@ -12,43 +12,43 @@ image: ../../static/img/bandwidth-logo.png
 
 ## Connecting WebRTC to Programmable Voice
 
-Bandwidth has a demonstrated history of excellence in IP voice technologies, including comprehensive CPaaS Programmable Voice and Messaging capabilities. The introduction of WebRTC provides the opportunity to combine the strength and diversity of our Programmable Voice with our highly controllable WebRTC to yield a rich and flexible platform that allows our customers to extend their business with a broad range of communication capabilities.
+Bandwidth has a demonstrated history of excellence in IP voice technologies, including comprehensive CPaaS Programmable Voice (PV) and Messaging capabilities. The introduction of WebRTC provides the opportunity to combine the strength and diversity of our Programmable Voice (PV) offering with our highly controllable WebRTC to yield a rich and flexible platform that allows our customers to extend their business with a broad range of browser-based communication capabilities.
```

Comment:
The second sentence is a little wordy IMO. How about:

Combining WebRTC with our Programmable Voice (PV) offering yields a rich and flexible platform that allows our customers to extend their business with a broad range of browser-based communication capabilities.